compiler/dspash/hdfs_file_data.py

```diff
@@ -32,51 +32,48 @@ def paths(self):
             )
         return filepaths
 
+    def _getinfo(self, filename):
```

Comment:
It is a bit scary (and unexpected) that when calling the initialization of an object we perform a subprocess call :O. Maybe that can happen in a separate function that is clearly called before initialization, in order to make it clear that here we perform a subprocess call, which could fail, etc...

Reply:
Ok, I could restructure this to be a function call with the object as a return.

Reply:
Sounds good